index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="icon" type="image/x-icon" href="./images/favicon-32x32.png">
+    <link href="https://fonts.googleapis.com/css2?family=Fraunces:opsz,wght@9..144,700&family=Montserrat:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;0,900;1,200;1,400;1,600;1,900&display=swap" rel="stylesheet">
+    <title>Product preview card</title>
+</head>
+<body>
+    <div class="cardContainer">
+        <div class="imageContainer"></div>
```

Comment:
w przypadku, gdy wstawiamy jakieś zdjęcie używa się standardowo 'img' a w przypadku, gdy nadajemy tło całej stronie to wtedy używa się background-image. Chodzi o to, że to zdjęcie stanowi część sekcji, która jest wyświetlana (czyli dotyczy tego produktu) - w takim przypadku powinno się je dodać jako img żeby też dodać atrybut 'alt', który wyświetli się w przypadku nieprawidłowego załadowania obrazka

Reply:
tutaj dokładny opis:
https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image